clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -149,14 +167,14 @@ class DockerRegistryClient {
     this.catalogFile = catalogFile
   }
 
-  DockerRegistryClient(String address, String email, String username, String password, long clientTimeoutMillis, int paginateSize, String catalogFile) {
-    this(address, clientTimeoutMillis, paginateSize)
+  DockerRegistryClient(String address, String email, String username, String password, long clientTimeoutMillis, int paginateSize, String catalogFile, boolean insecureRegistry) {
+    this(address, clientTimeoutMillis, paginateSize, catalogFile, insecureRegistry)
```

Comment:
I think f70d6ea introduces a potential bug as it's not passing catalogFile to the underlying constructor.

Reply:
@podnov I just merged this last night. I'll double check that. If it's missing, it was an oversight! Thanks for the heads up!

Reply:
@podnov fixed in #1903